---
layout: post
permalink: covid-19
published: true
listed: true
title: "Single-cell multi-omics analysis of the immune response in COVID-19"
date: 2021-04-20 00:00:00 +0000
categories: 
contact: Emily Stephenson
contact_email: emily.Stephenson2@newcastle.ac.uk
doi: 10.1038/s41591-021-01329-2
doi_link: https://doi.org/10.1038/s41591-021-01329-2
lab: Haniffa Lab
lab_website: https://haniffalab.com/
raw_data_link: http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-10026/
code_repository_link: https://github.com/scCOVID-19/COVIDPBMC
reproducibility_link:
---
Analysis of human blood immune cells provides insights into the coordinated response to viral infections such as severe acute respiratory syndrome coronavirus 2, which causes coronavirus disease 2019 (COVID-19). We performed single-cell transcriptome, surface proteome and T and B lymphocyte antigen receptor analyses of over 780,000 peripheral blood mononuclear cells from a cross-sectional cohort of 130 patients with varying severities of COVID-19. We identified expansion of nonclassical monocytes expressing complement transcripts (CD16+C1QA/B/C+) that sequester platelets and were predicted to replenish the alveolar macrophage pool in COVID-19. Early, uncommitted CD34+ hematopoietic stem/progenitor cells were primed toward megakaryopoiesis, accompanied by expanded megakaryocyte-committed progenitors and increased platelet activation. Clonally expanded CD8+ T cells and an increased ratio of CD8+ effector T cells to effector memory T cells characterized severe disease, while circulating follicular helper T cells accompanied mild disease. We observed a relative loss of IgA2 in symptomatic disease despite an overall expansion of plasmablasts and plasma cells. Our study highlights the coordinated immune response that contributes to COVID-19 pathogenesis and reveals discrete cellular components that can be targeted for therapy.